CMakeLists.txt

```diff
@@ -211,13 +230,13 @@ set(INCLUDE_DIRS
     ${CMAKE_SOURCE_DIR}/src/sip/dynamic_data;
     ${CMAKE_SOURCE_DIR}/src/sip/mpi;
     ${CMAKE_SOURCE_DIR}/src/sip/cuda;
-    ${CMAKE_SOURCE_DIR}/jsoncpp;
+    ${ACES_JASON_DIR};
```

Comment:
Haha nice catch.  I think json is turned off because it crappy out on build
on Cray's.
On May 17, 2016 5:02 PM, "Nakul Jindal" <notifications@github.com> wrote:

> In CMakeLists.txt
> <https://github.com/UFParLab/aces4/pull/71#discussion_r63603270>:
>
> > @@ -211,13 +230,13 @@ set(INCLUDE_DIRS
> >      ${CMAKE_SOURCE_DIR}/src/sip/dynamic_data;
> >      ${CMAKE_SOURCE_DIR}/src/sip/mpi;
> >      ${CMAKE_SOURCE_DIR}/src/sip/cuda;
> > -    ${CMAKE_SOURCE_DIR}/jsoncpp;
> > +    ${ACES_JASON_DIR};
>
> ACES_JASON_DIR -> ACES_JSON_DIR
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/UFParLab/aces4/pull/71/files/45c566834241cb0bdbf66c3bf0b92d517fa6dc31#r63603270>
>